acvm/src/pwg.rs

```diff
@@ -30,37 +42,31 @@ pub fn witness_to_value(
         None => Err(OpcodeNotSolvable::MissingAssignment(witness.0).into()),
     }
 }
+
+pub fn expression_to_const(expr: &Expression) -> Option<FieldElement> {
```

Comment:
Lets put this as a method on `Expression`:
```rust
impl Expression{

   pub fn to_const(&self) -> Option<FieldElement>{
      self.is_const().then_some(self.q_c)
   } 

}
```

Reply:
Done